config/formats.js

```diff
@@ -562,6 +562,38 @@ let Formats = [
 		ruleset: ['Pokemon', 'Standard NEXT', 'Team Preview'],
 		banlist: ['Uber'],
 	},
+	{
+		name: '[Gen 7] Metronome Battle',
+		threads: [
+			`&bullet; <a href="http://spo.ink/shitmons">Metronome Battle</a>`,
+		],
+
+		mod: 'gen7',
+		gameType: 'doubles',
+		forcedLevel: 666,
+		teamLength: {
+			validate: [2, 2],
+			battle: 2,
+		},
+		ruleset: ['Cancel Mod', 'HP Percentage Mod'],
+		banlist: [
+			'Honedge', 'Doublade', 'Aegislash', 'Kitsunoh', 'Shedinja + Sturdy', 'Power Construct', 'Schooling', 'Iron Barbs', 'Rough Skin',
+			'Pure Power', 'Huge Power', 'Cursed Body', 'Moody', 'Pressure', 'Wonder Guard', 'Fur Coat', 'Parental Bond', 'Protean', 'Stamina',
+			'Snow Warning', 'Sand Stream', 'Primordial Sea', 'Desolate Land', 'Poison Heal', 'Ice Body', 'Rain Dish', 'Grassy Surge',
+			'Cheek Pouch', 'Aguav Berry', 'Berry', 'Berry Juice', 'Black Sludge', 'Enigma Berry', 'Figy Berry', 'Iapapa Berry', 'Leftovers',
+			'Mago Berry', 'Oran Berry', 'Sitrus Berry', 'Wiki Berry', 'Rocky Helmet', 'Normalium Z', 'Aerodactylite', 'Aggronite', 'Ampharosite',
+			'Blastoisinite', 'Blazikenite', 'Charizardite X', 'Charizardite Y', 'Diancite', 'Galladite', 'Garchompite', 'Gardevoirite',
+			'Gyaradosite', 'Latiasite', 'Latiosite', 'Lucarionite', 'Metagrossite', 'Salamencite', 'Sceptilite', 'Steelixite', 'Swampertite',
+			'Tyranitarite', 'Venusaurite',
+		],
+		onValidateSet: function (set) {
+			let template = this.getTemplate(set.species);
+			if (set.name === template.baseSpecies) return [`${set.species} does not have a nickname set.`, `(Edgy nicknames are required)`];
+			let bst = template.baseStats.hp + template.baseStats.atk + template.baseStats.def + template.baseStats.spa + template.baseStats.spd + template.baseStats.spe;			
+			if (bst > 600) return [`${template.species} has a too high BST.`, `(Pok\u00e9mon with a BST higher than 600 are banned)`];
```

Comment:
The main reason why I said "I feel like the wording was better" was because I didn't want to be rude. This sounds clumpy and is actually grammatically incorrect unless you hyphenate "too-high", yet after that it would still sound wonky.

Reply:
...then just say it's grammatically incorrect and explain why :l